settings/settings.json

```
@@ -110,7 +111,7 @@
                 "children": [
                     {
                         "title": "Hide Clock",
-                        "description": "Hides all clocks in Jellyfin. Jellyfin will need to be closed and reopened for change to take effect.",
+                        "description": "Hide all clocks in Jellyfin. Jellyfin will need to be closed and reopened for change to take effect.",
```

```suggestion
                        "description": "Hide all clocks in Jellyfin. Jellyfin will need to be closed and reopened for changes to take effect.",
```

I'm no English major but "changes" sounds better and since we are hiding "all clocks" that implies there are multiple changes. I think 😄 

If you do change this line, we'll need to modify the translation string as well

---

lol i'm not sure why this changed, i agree with you.